docs/Sonarr/sonarr-collection-of-custom-formats.md

```diff
@@ -994,6 +994,25 @@ We've made 3 guides related to this.
 
 ---
 
+### FreeLeech
+
+??? question "FreeLeech - [Click to show/hide]"
+
+    Sometimes, torrent sites set a torrent to be freeleech. This means, that the download of this torrent will not count towards your download quota or ratio. This is really useful, if you do not have the best ratio yet.
```

Comment:
```suggestion
    Torrent sites sometimes set a torrent to be freeleech. This means that the download of this torrent will not count towards your quota or ratio. This can be useful when starting a new account or if you do not have the best ratio yet.